packages/products-used/style.module.css

```diff
@@ -0,0 +1,84 @@
+.root {
+  &.dark {
+    background-color: var(--wpl-neutral-900);
+  }
+}
+
+.eyebrow {
+  composes: g-type-label-strong from global;
+  margin: 0 0 var(--wpl-spacing-03) 0;
+  color: var(--wpl-neutral-500);
+
+  &.dark {
+    color: var(--wpl-neutral-400);
+  }
+}
+
+.productList {
+  display: flex;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 0;
+  list-style-type: none;
+  gap: var(--wpl-spacing-03);
+
+  &.isSmallerGap {
+    gap: var(--wpl-spacing-01);
+  }
+}
+
+.product {
+  display: flex;
+  align-items: center;
+
+  &[href] {
+    padding: var(--wpl-spacing-01) var(--wpl-spacing-02);
+    border-radius: calc(var(--wpl-spacing-01) / 2);
+    border: 1.5px solid var(--wpl-neutral-100);
```

Comment:
Always tend to be hesitant to use half pixels as values. What do yo think?

Reply:
Same - bumped to 2px.